backend/compact-connect/docs/it_staff_onboarding_instructions.md

```diff
@@ -157,8 +158,66 @@ Note: The URL was provided during onboarding and is already configured for your
 In addition to calling the POST endpoint, there is also an option to upload license data in a CSV file format.
 This method may be preferable for larger datasets or for systems that already generate CSV exports.
 
-For detailed documentation on CSV file uploads, including required fields, formatting requirements, and the upload
-process, please refer to [the technical user guide](./README.md#machine-to-machine-automated-uploads).
+#### CSV Upload Process
```

Comment:
Thanks for adding these details. I think the full walk-through makes more sense here than it does in the original docs/README.md, now that we have a more specific file for this. Can we remove the duplicate content from `docs/README.md` and link over here for details?